app/src/main/java/org/simple/clinic/summary/bloodsugar/view/BloodSugarSummaryView.kt

```diff
@@ -292,7 +298,7 @@ class BloodSugarSummaryView(
   }
 
   private fun isBloodSugarEditable(measurement: BloodSugarMeasurement): Boolean {
-    if (bloodSugarSummaryConfig.bloodSugarEditFeatureEnabled.not() || measurement.reading.type is Unknown) {
+    if (features.isEnabled(EditBloodSugar).not() || measurement.reading.type is Unknown) {
```

Comment:
Since we have `Features#isDisabled` should we use that? WDYT?

Reply:
Good catch, I introduced that method to handle this case, but it looks like I forgot to use it 😓